source/includes/steps-create-system-dsn-macos.yaml

```diff
@@ -76,108 +88,153 @@ content: |
 title: (Optional) If authentication is enabled, add the authentication keywords.
```

Comment:
If we're going to change some steps to "settings" we'll need to change it here. Not sure if I really like that change.

Reply:
Kept `keywords`.